dvc/command/experiments.py

```diff
@@ -27,32 +27,23 @@ def _filter_name(names, label, filter_strs):
 
     for filter_s in filter_strs:
         path, _, name = filter_s.rpartition(":")
-        path_filters[path].append(tuple(name.split(".")))
+        path_filters[path].append(name)
```

Comment:
We should also add a note in the docs PR that by default, filter patterns will be applied to all params/metrics files, unless the filter specifically includes the `path:` prefix, and that wildcards are not supported in the path prefix itself (the file path must be explicit)

Reply:
Hm, I guess I was thinking it would be a match against the full column header (`path:section.param` where paths are included) since this would be most intuitive for users. Seems like this is halfway there, matching everything after the path, which as you note requires some additional documentation.

Reply:
@dberenbaum If I have a repo with 2 parameter files:

```yaml
# params.yaml
foo: 1
foobar: 2
```

```yaml
# other_params.yaml
foo: 3
```
The table will contain 3 columns: `params.yaml:foo`, `foobar`, `other_params.yaml:foo`

With the current implementation, `foo*` will match all 3 columns. With the proposed change, you would need `*foo*` to match all 3 columns.

The first pattern seems most intuitive to me, especially since in DVC we default to omitting pathnames for params/metrics whenever possible, but I'm not particularly opposed to going with the alternative either.

Reply:
I know this is kind of a contrived example, but "params with matching prefixes in different files" does seem like a realistic/potential user scenario to me.

Reply:
Yeah, thanks for the example. Since dvc omits path names where possible, it's a little confusing regardless of the choice. My suggestion would make it really hard to get all the parameters from a single file, which seems like enough reason to discard it.

What about doing fnmatch separately on the path and the param, assuming the `:` is still needed if including a path, so that `p*:*` would include the first two columns? @pmrowla @daavoo up to you at this point.

Reply:
I'm in favor of leaving this as-is for simplicities sake, if/when enough people ask for the additional matching we can address it at that time.

Reply:
Ok!